meta/runtime.yml

```diff
@@ -4305,6 +4305,20 @@ plugin_routing:
         warning_text: You are using an internal name to access the community.general.wakeonlan
           modules. This has never been supported or documented, and will stop working
           in community.general 9.0.0.
+    remote_management.ocapi.ocapi_command:
+      redirect: community.general.ocapi_command
+      deprecation:
+        removal_version: 9.0.0
+        warning_text: You are using an internal name to access the community.general.ocapi_command
+          modules. This has never been supported or documented, and will stop working
+          in community.general 9.0.0.
+    remote_management.ocapi.ocapi_info:
+      redirect: community.general.ocapi_info
+      deprecation:
+        removal_version: 9.0.0
+        warning_text: You are using an internal name to access the community.general.ocapi_info
+          modules. This has never been supported or documented, and will stop working
+          in community.general 9.0.0.
```

Comment:
Please remove all changes from this file. New modules must not be added here.

Reply:
Removed.